packages/rollup-core/test/app/block-batch-processor.spec.ts

```diff
@@ -86,6 +87,7 @@ const getTransactionResponse = (
   timestamp: number,
   data: string,
   hash: string,
+  _gasLimit: number,
```

Comment:
just noting there's a `_`, maybe that's a convention you want though.

Reply:
There's a `gasLimit` file-scoped const that this would shadow and our linter doesn't like that, so I just made it `_gasLimit`